packages/react-slider/Spec.md

```diff
@@ -1,63 +1,230 @@
+| origin       | x                                                                                       | x                                                                                     | The starting origin point for the **Slider**. @defaultValue (renders at 0)                                                                                                   |
+| min          | &check;                                                                                 | &check;                                                                               | The min value of the **Slider**.                                                                                                                                             |
+| max          | &check;                                                                                 | &check;                                                                               | The max value of the **Slider**.                                                                                                                                             |
+| step         | &check;                                                                                 | &check;                                                                               | The number of steps that the **Slider's** `value` will increment upon change. When passing the string "marks" **Slider** will step to adjacent values from the `marks` prop. |
+
+### Visual behavior props
+
+| Name              | <img src="https://img.shields.io/badge/Used%20in-v0-orange" alt="drawing" width="200"/> | <img src="https://img.shields.io/badge/Used%20in-v8-blue" alt="drawing" width="200"/> | Description                                                                                                   |
+| ----------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------- |
+| label             | x                                                                                       | &check;                                                                               | The description label of the **Slider**.                                                                      |
+| valueLabel        | x                                                                                       | &check;                                                                               | The current value or unique format to be shown for the **Slider's** `value label`.                            |
```

Comment:
I'm wondering if this should be a function that takes the value and returns a string, since if someone wants custom value text, they probably also want it for the min/max values.

If that's already your plan, then ignore this 😄

Reply:
What I was thinking for the `valueLabel` prop is that it would handle the two previous API features: `valueFormat` and `showValue`.
If someone wanted to render a valueLabel they could:
1. Pass the Boolean `true` and receive the currentValue.
2. Pass a method like this to customize it:
```jsx
const sliderValue = (value: number) => `${value}%`;
```

Reply:
Got it, thanks for the explanation! I still wonder if it might be better to split those out though. For example, what if someone wanted to customize the min/max labels + maybe the tick labels, but not show separate text for the current value?